### PR TITLE
Fix output of `anoma wallet address find`

### PR DESCRIPTION
**Pull request by brentstone**
Fixes [#993](https://github.com/anoma/namada/issues/91). Based and depends on anoma/anoma#1138 

 The output is now:

image

### CODE REVIEW

**Comment by Fraccaman** [comment]
I think `--alias` should be optional as we might introduce other arguments (like --address, which would do the opposite)

**Comment by brentstone** [comment]
> I think `--alias` should be optional as we might introduce other arguments (like --address, which would do the opposite). I think this should be `at least one arg` (using clap `ArgGroup`)

In this particular case, would we want `--address` to return the `alias`, or should searching for aliases be its own new subcommand?

**Comment by Fraccaman** [comment]
I think `--address` should return the alias

File: apps/src/lib/cli/utils.rs [hunk 1]
```diff
@@ -170,6 +170,7 @@ where
     <T as FromStr>::Err: Debug,
 {
     pub fn parse(&self, matches: &ArgMatches) -> T {
+        println!("I'm parsing!");
```
Comment (Fraccaman) [inline apps/src/lib/cli/utils.rs:173]: I think we can remove this log

Comment (Fraccaman) [inline apps/src/lib/cli/utils.rs:173]: I think we can remove this log

**Review by Fraccaman — DISMISSED** [review]

**Comment by Fraccaman** [comment]
Looking great :) requires changelog

**Comment by brentstone** [comment]
CI build is now failing on e2e tests after only removing a println! statement and adding the changelog. Is it possible something else changed in the CI? Not sure how the last two commits could have changed the behavior of the build.

**Comment by tzemanovic** [comment]
> CI build is now failing on e2e tests after only removing a println! statement and adding the changelog. Is it possible something else changed in the CI? Not sure how the last two commits could have changed the behavior of the build.

Yeah, there seem to be some flakiness with e2e tests (possibly caused by expectrl switch #1095) and occasionally sccache timeouts :/

**Comment by tzemanovic** [comment]
moved to https://github.com/anoma/namada/pull/276

**Pull request closed without merging**